.storybook/test-runner.cjs

```diff
@@ -0,0 +1,35 @@
+const { injectAxe, checkA11y } = require('axe-playwright');
+
+const config = {
+  runOnly: {
+    type: 'tag',
+    values: ['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa']
+  }
+};
+
+/*
+* See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api-experimental
+* to learn more about the test-runner hooks API.
+*/
+module.exports = {
+  async preRender(page) {
+    await injectAxe(page);
+  },
+  async postRender(page, context) {
+    const axeOptions = context.name === 'Loading'
```

Comment:
Does this mean that the name of the view must be exactly "Loading" in order for it to apply the loading config?  I'm not sure if there would be a situation where we would have more than one loading view for a story, but would it be good to make it so that it'll apply this config if "loading" or "Loading" appears anywhere in the name?

Reply:
Yes, the story name would need to be an exact match. I thought about being more generic, but I'd prefer to err on the side of caution and only disable rules for the fewest cases right now. If we start writing other loading stories with different naming conventions, we could easily expand this matching then